erpnext/hr/doctype/attendance/attendance.json

```
@@ -199,13 +193,60 @@
   {
    "fieldname": "column_break_18",
    "fieldtype": "Column Break"
+  },
+  {
+   "depends_on": "overtime_type",
+   "fieldname": "overtime_details_section",
+   "fieldtype": "Section Break",
+   "label": "Overtime Details"
+  },
+  {
+   "depends_on": "overtime_type",
+   "fieldname": "overtime_type",
+   "fieldtype": "Link",
+   "label": "Overtime Type",
+   "options": "Overtime Type",
+   "read_only": 1
+  },
+  {
+   "description": "Shift duration for a day",
+   "fetch_from": "shift.standard_working_time",
+   "fieldname": "standard_working_time",
+   "fieldtype": "Duration",
+   "label": " Standard Working Time",
+   "read_only": 1
```

```suggestion
   "fetch_from": "shift.standard_working_time",
   "fieldname": "shift_duration",
   "fieldtype": "Duration",
   "label": "Shift Duration",
   "read_only": 1
```
Should we call it Shift Duration and remove the description instead?

---

Done